sites/main-site/src/content/events/2024/hackathon-barcelona.mdx

```diff
@@ -143,6 +143,13 @@ You can move freely between projects throughout the event.
 
 </HackathonGroup>
 
+<HackathonGroup title="nf-test plugins" color="#238636" leaders={{'nvnieuwk': 'Nicolas Vannieuwkerke'}}>
+    _Details coming soon_
```

Comment:
Can you add some details please? Doesn't need to much, my intention was to use the forms in slack for this so you've already written something. I only wrote this for the other projects because I added them without first talking to their group leads 😅 

Suggestion here, with wording tweaked slightly:

```suggestion
    nf-test is a very important piece of our modules, used for continuous integration testing of all our modules.
    However, writing tests for some file types / more advanced tests can be difficult.
    In this group we will try and kickstart the creation of nf-test plugins to make our testing a lot easier.
    This will mainly involve the development of [`nft-utils`](https://github.com/nf-core/nft-utils), the improvement of [`nft-bam`](https://nvnieuwk.github.io/nft-bam/latest/) and hopefully the creation of completely new nf-test plugins.
```